<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabela_TarefaFuncionario_2</class>
 <widget class="QMainWindow" name="tabela_TarefaFuncionario_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 137, 206);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_User">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Olá {user.nome}  Hoje é dia {datetime.today()}, {datetime.dayToday} </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabela_TarefaFuncionario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(243, 243, 243);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <column>
     <property name="text">
      <string>Tarefa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prazo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prioridade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="botao_Att">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Atualização</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_Saida">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Saída</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_Entrada">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Entrada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_Contato">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Contato</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
